utils/create_srg_export.py

```diff
@@ -0,0 +1,203 @@
+                html_plain_text(srg.find('xccdf-1.1:check/xccdf-1.1:check-content', NS).text)
+            srgs[srg_id]['ia_controls'] = description_root.find('IAControls').text
+    return srgs
+
+
+def handle_rule_yaml(product: str, rule_dir: str, env_yaml: dict) -> ssg.build_yaml.Rule:
+    rule_file = ssg.rules.get_rule_dir_yaml(rule_dir)
+
+    rule_yaml = ssg.build_yaml.Rule.from_yaml(rule_file, env_yaml=env_yaml)
+    rule_yaml.normalize(product)
+
+    return rule_yaml
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-c', '--control', type=str, action="store", required=True,
+                        help="The control file to parse")
+    parser.add_argument('-o', '--output', type=str, help="The path to the output",
+                        default=OUTPUT)
+    parser.add_argument("-r", "--root", type=str, action="store", default=SSG_ROOT,
+                        help=f"Path to SSG root directory (defaults to {SSG_ROOT})")
+    parser.add_argument("-j", "--json", type=str, action="store", default=RULES_JSON,
+                        help="Path to the rules_dir.json (defaults to build/stig_control.json)")
+    parser.add_argument("-p", "--product", type=str, action="store", required=True,
+                        help="What product to get STIGs for")
+    parser.add_argument("-b", "--build-config-yaml", default=BUILD_CONFIG,
+                        help="YAML file with information about the build configuration. ")
+    parser.add_argument("-m", "--manual", type=str, action="store",
+                        help="Path to XML XCCDF manual file to use as the source of the SRGs",
+                        default=SRG_PATH)
+    return parser.parse_args()
+
+
+def handle_control(product: str, control: ssg.controls.Control, csv_writer: csv.DictWriter,
+                   env_yaml: ssg.environment, item: dict, rule_json: dict, srgs: dict) -> None:
+    if control.selections:
+        for rule in control.selections:
+            row = create_base_row(item, srgs)
+            rule_object = handle_rule_yaml(product, rule_json[rule]['dir'], env_yaml)
+            row['Requirement'] = rule_object.description
+            row['Vul Discussion'] = rule_object.rationale
+            row['Check'] = rule_object.ocil
+            row['Fix'] = rule_object.fix
+            # If then control has rule by definition the status is "Applicable - Configurable"
+            row['Status'] = DisaStatus.AUTOMATED
+            csv_writer.writerow(row)
+    else:
+        row = create_base_row(item, srgs)
+        row['Mitigation'] = control.mitigation
+        row['Artifact Description'] = control.artifact_description
+        row['Status Justification'] = control.status_justification
+        row['Status'] = DisaStatus.from_string(control.status)
+        csv_writer.writerow(row)
+
+
+def create_base_row(item, srgs):
+    row = dict()
+    srg_id = item['id']
+    srg = srgs[srg_id]
```

Comment:
What if the ID isn't present in `srgs`?

Reply:
The script now errors out in a nice way.